js/main.js

```diff
@@ -25,4 +33,61 @@ Pineapples can be consumed fresh, cooked, juiced, or preserved. They are found i
 
 Strawberry
 The garden strawberry (or simply strawberry; Fragaria × ananassa) is a widely grown hybrid species of the genus Fragaria (collectively known as the strawberries). It is cultivated worldwide for its fruit. The fruit (which is not a botanical berry, but an aggregate accessory fruit) is widely appreciated for its characteristic aroma, bright red color, juicy texture, and sweetness. It is consumed in large quantities, either fresh or in such prepared foods as preserves, fruit juice, pies, ice creams, milkshakes, and chocolates. Artificial strawberry flavorings and aromas are also widely used in many products like lip gloss, candy, hand sanitizers, perfume, and many others.
-*/
\ No newline at end of file
+*/
+
+//this is my code//
+$(document).ready(function() {
+	
+
+ ///var fruit = $('fruit li').val();
+
+		//$('.fruit li').val();     
+       
+
+ 
+	 $('.fruit li').click(function() {
+	 		event.preventDefault();
```

Comment:
`event.preventDefault()` isn't actually needed here. There is no _default_ action for an `li` so you can omit this. Also, just a heads up, if you are gonna include `event.preventDefault()`, you need to pass the `event` object as a parameter into the event handler function on line 48